### ISSUE TITLE
Cannot create new when saying yes to SSR

### ISSUE THREAD

**Issue by tombohub** [angular/angular-cli#26745]
### Which @angular/* package(s) are the source of the bug?

core

### Is this a regression?

Yes

### Description

Trying to create `ng new` project. 

select SCSS
select SSR - yes

gives error:

```
✖ Package install failed, see above.
The Schematic workflow failed. See above.
```

full text:

```
 Which stylesheet format would you like to use? SCSS   [ https://sass-lang.com/documentation/syntax#scss
           ]
? Do you want to enable Server-Side Rendering (SSR) and Static Site Generation (SSG/Prerendering)? Yes
CREATE sas-frontend3/angular.json (2947 bytes)
CREATE sas-frontend3/package.json (1284 bytes)
CREATE sas-frontend3/README.md (1066 bytes)
CREATE sas-frontend3/tsconfig.json (903 bytes)
CREATE sas-frontend3/.editorconfig (274 bytes)
CREATE sas-frontend3/.gitignore (548 bytes)
CREATE sas-frontend3/tsconfig.app.json (324 bytes)
CREATE sas-frontend3/tsconfig.spec.json (273 bytes)
CREATE sas-frontend3/server.ts (1703 bytes)
CREATE sas-frontend3/.vscode/extensions.json (130 bytes)
CREATE sas-frontend3/.vscode/launch.json (470 bytes)
CREATE sas-frontend3/.vscode/tasks.json (938 bytes)
CREATE sas-frontend3/src/main.ts (250 bytes)
CREATE sas-frontend3/src/favicon.ico (15086 bytes)
CREATE sas-frontend3/src/index.html (298 bytes)
CREATE sas-frontend3/src/styles.scss (80 bytes)
CREATE sas-frontend3/src/main.server.ts (264 bytes)
CREATE sas-frontend3/src/app/app.component.html (20884 bytes)
CREATE sas-frontend3/src/app/app.component.spec.ts (937 bytes)
CREATE sas-frontend3/src/app/app.component.ts (372 bytes)
CREATE sas-frontend3/src/app/app.component.scss (0 bytes)
CREATE sas-frontend3/src/app/app.config.ts (321 bytes)
CREATE sas-frontend3/src/app/app.routes.ts (77 bytes)
CREATE sas-frontend3/src/app/app.config.server.ts (350 bytes)
CREATE sas-frontend3/src/assets/.gitkeep (0 bytes)
✔ Packages installed successfully.
warning: LF will be replaced by CRLF in .editorconfig.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in .gitignore.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in .vscode/extensions.json.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in .vscode/launch.json.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in .vscode/tasks.json.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in README.md.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in angular.json.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in package.json.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in pnpm-lock.yaml.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in server.ts.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in src/app/app.component.html.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in src/app/app.component.spec.ts.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in src/app/app.component.ts.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in src/app/app.config.server.ts.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in src/app/app.config.ts.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in src/app/app.routes.ts.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in src/index.html.
warning: LF will be replaced by CRLF in src/main.server.ts.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in src/main.ts.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in src/styles.scss.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in tsconfig.app.json.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in tsconfig.json.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in tsconfig.spec.json.
The file will have its original line endings in your working directory
    Successfully initialized git.
✖ Package install failed, see above.
The Schematic workflow failed. See above.
```

### Please provide the exception or error you saw

```true
Successfully initialized git.
✖ Package install failed, see above.
The Schematic workflow failed. See above.
```

### Please provide the environment you discovered this bug in (run `ng version`)

```true
_                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 17.0.8
Node: 18.15.0
Package Manager: pnpm 8.6.10
OS: win32 x64

Angular:
...

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.1700.8 (cli-only)
@angular-devkit/core         17.0.8 (cli-only)
@angular-devkit/schematics   17.0.8 (cli-only)
@schematics/angular          17.0.8 (cli-only)
```

**Comment by alan-agius4** [comment]
Hi @tombohub, unfortunately I am unable to replicate this problem. Can you try updating PNPM to the latest version?

**Comment by tombohub** [comment]
Hi @alan-agius4 , I still get the same error:

```
    Successfully initialized git.
✖ Package install failed, see above.
The Schematic workflow failed. See above.
```

ng version:

```

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 17.0.8
Node: 18.15.0
Package Manager: pnpm 8.13.1
OS: win32 x64

Angular:
...

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.1700.8 (cli-only)
@angular-devkit/core         17.0.8 (cli-only)
@angular-devkit/schematics   17.0.8 (cli-only)
@schematics/angular          17.0.8 (cli-only)
```

Is there anything else I can do?

**Comment by alan-agius4** [comment]
@tombohub, if you run `pnpm install` directly in the workspace do you get additional information?

**Comment by tombohub** [comment]
@alan-agius4 everything seems normal

pnpm install:

```
PS D:\PROJEKTI\prototypes\angagit> pnpm install
Lockfile is up to date, resolution step is skipped
Already up to date
Done in 1.2s
```

ng serve:
```
PS D:\PROJEKTI\prototypes\angagit> ng serve

Initial Chunk Files | Names         |  Raw Size
polyfills.js        | polyfills     |  82.74 kB | 
main.js             | main          |  23.32 kB | 
styles.css          | styles        |  96 bytes | 

                    | Initial Total | 106.15 kB

Application bundle generation complete. [8.252 seconds]
Watch mode enabled. Watching for file changes...
  ➜  Local:   http://localhost:4200/
```

ng build:
```
PS D:\PROJEKTI\prototypes\angagit> ng build    
Prerendered 1 static route.


Initial Chunk Files   | Names         |  Raw Size | Estimated Transfer Size
main-4I5U5YQS.js      | main          | 203.78 kB |                56.29 kB
polyfills-TMVK3KFA.js | polyfills     |  32.69 kB |                10.59 kB
styles-5INURTSO.css   | styles        |   0 bytes |                 0 bytes

                      | Initial Total | 236.47 kB |                66.89 kB

Application bundle generation complete. [12.082 seconds]
```

**Comment by alan-agius4** [comment]
I suspect this should be solved in prerelease version of 17.1.x. 

Can you try using `pnpm dlx @angular/cli@next new`?

**Comment by tombohub** [comment]
@alan-agius4 `pnpm dlx @angular/cli@next new` command gives no errors:

```
? What name would you like to use for the new workspace and initial project? ngtest
? Which stylesheet format would you like to use? SCSS   [ https://sass-lang.com/documentation/syntax#scss
           ]
? Do you want to enable Server-Side Rendering (SSR) and Static Site Generation (SSG/Prerendering)? Yes
CREATE ngtest/angular.json (3018 bytes)
CREATE ngtest/package.json (1348 bytes)
CREATE ngtest/README.md (1092 bytes)
CREATE ngtest/tsconfig.json (936 bytes)
CREATE ngtest/.editorconfig (290 bytes)
CREATE ngtest/.gitignore (590 bytes)
CREATE ngtest/tsconfig.app.json (342 bytes)
CREATE ngtest/tsconfig.spec.json (287 bytes)
CREATE ngtest/server.ts (1759 bytes)
CREATE ngtest/.vscode/extensions.json (134 bytes)
CREATE ngtest/.vscode/launch.json (490 bytes)
CREATE ngtest/.vscode/tasks.json (980 bytes)
CREATE ngtest/src/main.ts (256 bytes)
CREATE ngtest/src/favicon.ico (15086 bytes)
CREATE ngtest/src/index.html (305 bytes)
CREATE ngtest/src/styles.scss (81 bytes)
CREATE ngtest/src/main.server.ts (271 bytes)
CREATE ngtest/src/app/app.component.html (21206 bytes)
CREATE ngtest/src/app/app.component.spec.ts (945 bytes)
CREATE ngtest/src/app/app.component.ts (316 bytes)
CREATE ngtest/src/app/app.component.scss (0 bytes)
CREATE ngtest/src/app/app.config.ts (329 bytes)
CREATE ngtest/src/app/app.routes.ts (80 bytes)
CREATE ngtest/src/app/app.config.server.ts (361 bytes)
CREATE ngtest/src/assets/.gitkeep (0 bytes)
✔ Packages installed successfully.
warning: LF will be replaced by CRLF in package.json.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in pnpm-lock.yaml.
The file will have its original line endings in your working directory
warning: LF will be replaced by CRLF in src/app/app.config.ts.
The file will have its original line endings in your working directory
    Successfully initialized git.
```

ng version:

```

     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/


Angular CLI: 17.0.8
Node: 18.15.0
Package Manager: pnpm 8.13.1
OS: win32 x64

Angular:
...

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.1700.8 (cli-only)
@angular-devkit/core         17.0.8 (cli-only)
@angular-devkit/schematics   17.0.8 (cli-only)
@schematics/angular          17.0.8 (cli-only)
```

**Comment by alan-agius4** [comment]
In that case, closing since the issue has been solved in the 17.1.0, which will be released as stable later this month.
